rct229/data_fns/table_G3_6_fns.py

```diff
@@ -0,0 +1,82 @@
+from rct229.data import data
+from rct229.data_fns.table_utils import find_osstd_table_entry
+from rct229.schema.config import ureg
+
+# This dictionary maps the ExteriorLightingAreas2019ASHRAE901TableG36 enumerations to
+# the corresponding building_exterior_type values in the file
+# ashrae_90_1_table_G3_6.json
+
+EXTERIOR_LIGHTING_AREA_ENUMERATION_TO_BUILDING_EXTERIOR_TYPE_MAP = {
+    "UNCOVERED_PARKING_LOTS_AND_DRIVES": "Uncovered parking lots and drives",
+    "WALKWAY_NARROW": "Walkway - narrow",
+    "WALKWAY_WIDE": "Walkway - wide",
+    "PLAZA_AREAS": "Plaza Areas",
+    "SPECIAL_FEATURE_AREAS": "Special Feature Areas",
+    "STAIRWAYS": "Stairways",
+    "MAIN_ENTRANCE_DOOR": "Main entrance door",
+    "OTHER_ENTRANCE_OR_EXIT_DOORS": "Other entrance or exit doors",
+    "EXTERIOR_CANOPIES": "Exterior canopies",
+    "OUTDOOR_SALES_OPEN_AREAS": "Outdoor sales - open areas",
+    "STREET_FRONTAGE": "Street frontage",
+    "BUILDING_FACADE": "Building facade",
+    "AUTOMATED_TELLER_MACHINES": "Automated teller machines",
+    "NIGHT_DEPOSITORIES": "Night depositories",
+    "ENTRANCE_AND_GATEHOUSE": "Entrance and gatehouses",
+    "EMERGENCY_VEHICLE_LOADING_AREA": "Emergency vehicle loading area",
+    "DRIVE_UP_WINDOWS_FAST_FOOD": "Drive-up windows at fast-food restaurants",
+    "PARKING_NEAR_24HR_RETAIL_ENTRANCES": "Parking near 24-hour retail entrances",
+}
+
+
+def table_G3_6_lookup(building_exterior_type):
```

Comment:
I think the argument should be `exterior_lighting_area` since it is an ExteriorLightingAreas2019ASHRAE901TableG36 enumeration value. And then the documentation should match that.

Reply:
Sure, I also changed the json to have "exterior_lighting_area", instead of "building_exterior_type". If you think thats okay, I'll go ahead and merge these changes.